### ISSUE TITLE
Flag unnecessary space between function signature and return type

### ISSUE THREAD

**Issue by vanniktech** [shyiko/ktlint#68]
I'd except the following code to get flagged since the scape before `:` is not needed:

```kotlin
interface Something {
  fun render(time: Long) : String
}
```

Instead, it should look like this:

```kotlin
interface Something {
  fun render(time: Long): String
}
```

Tested with `0.8.3` and `0.9.0`.

**Comment by shyiko** [comment]
@vanniktech Thank you. This is definitely one of the things that could improved in [SpacingAroundColonRule](https://github.com/shyiko/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/SpacingAroundColonRule.kt).

**Comment by JLLeitschuh** [comment]
The formatter that `ktlint` suggests for intellij by default reformats the code to be:
```kotlin
interface Something {
  fun render(time: Long): String
}
```
I suggest making this the default.
